src/app/search/search-filters/known-values.ts

```diff
@@ -29,6 +29,15 @@ const D2EventPredicateLookup = Object.fromEntries(
   ]),
 );
 
+const D2SourcesAliasLookup: Record<string, string> = {};
+for (const [source, sourceAttrs] of Object.entries(D2Sources)) {
+  if (sourceAttrs.aliases) {
+    for (const alias of sourceAttrs.aliases) {
+      D2SourcesAliasLookup[alias] = source;
+    }
+  }
+}
+
```

Comment:
there is probably a more efficient way to handle this

Reply:
I was thinking we could go through each source and insert a copy into the original object with the alias' key.

Reply:
I will leave that to you

Reply:
nevermind i figured it out last night while sleeping